src/QMCWaveFunctions/BsplineFactory/SplineC2ROMPTarget.cpp

```diff
@@ -866,7 +866,7 @@ void SplineC2ROMPTarget<ST>::mw_evaluateVGLandDetRatioGrads(const RefVectorWithL
             omptarget::min(last_cplx + omptarget::min(nComplexBands_local, last_cplx), requested_orb_size);
         ValueType ratio(0), grad_x(0), grad_y(0), grad_z(0);
         PRAGMA_OFFLOAD("omp parallel for reduction(+: ratio, grad_x, grad_y, grad_z)")
-        for (size_t j = first_real; j < last_real; j++)
+        for (int j = first_real; j < last_real; j++)
```

Comment:
Why int for just a couple?

Reply:
A lot of places already has `int`. There is no need to change. I don't feel the need of 100% consistency. `int` for signed and `unit32_t` for unsigned. 64bit to 32bit is critical. signed vs unsigned is not.